Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -99,6 +100,14 @@ async function run() {
         if(webAppUri) {
             tl.setVariable(webAppUri, publishingProfile.destinationAppUrl);
         }
+		
+		if(renameFilesFlag){
```

Comment:
this **if** block can move inside `if(utility.canUseWebDeploy(useWebDeploy)) ` , since we need to enable this option only for MSDeploy